New version: Yubico.Authenticator version 6.3.0

Dragon1573 opened the pull request:
Resolve #120935 .

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Have you tested your manifest in Windows Sandbox with `Tools/SandboxTest.ps1 -Manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122100)

Dragon1573 committed:
New version: Yubico.Authenticator version 6.3.0

mdanish-kh commented on manifests/y/Yubico/Authenticator/6.3.0/Yubico.Authenticator.locale.en-US.yaml:
```diff
@@ -0,0 +1,26 @@
+Author: Yubico AB
+PackageName: Yubico Authenticator
+PackageUrl: https://www.yubico.com/products/yubico-authenticator/
+License: BSD-2-Clause License
```
```suggestion
License: Apache-2.0 License
```

Dragon1573 committed:
Fixup: Change License

- The package is released under Apache-2.0 License
- Also change the "Coryright" character for readability
